README.md

```diff
@@ -68,6 +68,8 @@ Here are some useful hints to get involved.
 * Documentation is always hard, we have [some information](https://criu.org/Category:Empty_articles) that is to be extracted from people's heads into wiki pages as well as [some texts](https://criu.org/Category:Editor_help_needed) that all need to be converted into useful articles;
 * Feedback is expected on the github issues page and on the [mailing list](https://lists.openvz.org/mailman/listinfo/criu);
 * We accept github pull requests and this is the preferred way to contribute to CRIU. If you prefer to send patches by email, you are welcome to send them to [the devel list](http://criu.org/How_to_submit_patches);
+  * Please avoid force pushing updates to an existent pull request. Close the pull request and open a new one with the updated changes.
```

Comment:
>I do not propose to make a PR per version a strict requirement. And in
any rate it's about the convinience of *reviewers* rather than
contributors ;-)

Than maybe change it to something like:

```
* Please avoid pushing fixup commits to an existent pull request, each commit should be self consistent and patch series shouldn't contain one commit which breaks something and other commit which fixes that something, any fix to the pull request should be merged in the commit from this pull request which introduces the problem.
* Introducing changes to pull requests can be done with closing old pull request and opening a fresh new one with the updated changes and with mandatory link to old pull request in the description. Also this can be done updating current pull request with force-pushing, but this mostly applies only to small changes and you should know what you are doing =).
```

Reply:
Well, that sounds reasonable :)
Since I've already opened a PR #1100 to add an elaborate CONTRIBUTING.md, let's wait for it to get merged and then put PR guidelines there.